testapi.pm

```diff
@@ -244,19 +244,19 @@ sub _check_backend_response {
 }
 
 sub _check_or_assert {
-    my ($mustmatch, $timeout, $check) = @_;
+    my ($mustmatch, $timeout, $check, %args) = @_;
     $timeout = bmwqemu::scale_timeout($timeout);
 
     die "current_test undefined" unless $autotest::current_test;
 
-    my $rsp = query_isotovideo('check_screen', {mustmatch => $mustmatch, timeout => $timeout, check => $check});
+    my $rsp = query_isotovideo('check_screen', {mustmatch => $mustmatch, timeout => $timeout, check => $check, interval => $args{interval}});
     # seperate function because it needs to call itself
-    return _check_backend_response($rsp, $check, $timeout, $mustmatch);
+    return _check_backend_response($rsp, $check, $timeout, $mustmatch, $args{interval});
 }
 
 =head2 assert_screen
 
-  assert_screen($mustmatch [,$timeout]);
+  assert_screen($mustmatch [,$timeout] [, interval => $interval]);
```

Comment:
if you extend the API, then please also support 'mustmatch', timeout => 30, interval => 1

I don't think we need to deprecate the simple form - as it's just a very frequent call